src/cms/views/dashboard/admin_dashboard_view.py

```diff
@@ -34,6 +37,18 @@ def get(self, request, *args, **kwargs):
         :return: The rendered template response
         :rtype: ~django.template.response.TemplateResponse
         """
+        all_feedback = Feedback.objects.filter(is_technical=True)[:5]
+        admin_feedback_exists = Feedback.objects.filter(is_technical=True).exists
 
         val = "To be defined"
-        return render(request, self.template_name, {"key": val, **self.base_context})
+        return render(
+            request,
+            self.template_name,
+            {
+                "key": val,
+                "current_menu_item": "admin_feedback",
+                "admin_feedback_exists": admin_feedback_exists,
```

Comment:
```suggestion
```
not required, see above

Reply:
there doesn't seem to be any space from me. everything is in order.

Reply:
The variable `admin_feedback_exists` is not used in the `admin_dashboard.html`. Therefore, you should remove it.